pkg/osquery/table/gdrive_sync.go

```diff
@@ -0,0 +1,61 @@
+func GDrivePlugin(client *osquery.ExtensionManagerClient) *table.Plugin {
```

Comment:
add the imports and package name.

Reply:
Thanks I just saw this was missing the sqlite import!

I thought the package name was supposed to be table? (per your example)